### PR TITLE
Fix Idn failing test

### PR DESCRIPTION
**Pull request by tarekgh**
This change i sreverting the change done in this test which tried to make it work with checking different platforms.
this didn't work well and expected will make the test not stable when we support more platforms.

### CODE REVIEW

**Comment by tarekgh** [comment]
cc @stephentoub @Clockwork-Muse

**Comment by tarekgh** [comment]
@dotnet-bot test innerloop Debian8.4 Debug
@dotnet-bot test innerloop Debian8.4 Release
@dotnet-bot test innerloop Fedora23 Debug
@dotnet-bot test innerloop Fedora23 Release
@dotnet-bot test innerloop OpenSUSE13.2 Debug
@dotnet-bot test innerloop OpenSUSE13.2 Release
@dotnet-bot test innerloop OpenSUSE42.1 Debug
@dotnet-bot test innerloop OpenSUSE42.1 Release
@dotnet-bot test outerloop Ubuntu16.04 Release
@dotnet-bot test outerloop Ubuntu16.10 Debug

**Comment by tarekgh** [comment]
the failure

```
Failure to restore a package: error : Exact package 'System.Security.Cryptography.OpenSSL 4.4.0-beta-24619-02' was not restored: found ''
```

is known failure and @joperezr is working on fixing it.

File: src/System.Globalization.Extensions/tests/IdnMapping/IdnMappingGetAsciiTests.cs [hunk 1]
```diff
@@ -87,19 +87,18 @@ public void GetAscii(string unicode, int index, int count, string expected)
...
+                // Windows and OSX always throw exception. some versions of Linux succeed and others throw exception
+                Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Windows));
+                Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.OSX));
+                Assert.Equal(result, ".");
             }
             else
             {
-                string result = mapping.GetAscii(dot);
-                Assert.Equal(dot, result);
+                Assert.True(ex is ArgumentException);
```
Comment (stephentoub) [inline src/System.Globalization.Extensions/tests/IdnMapping/IdnMappingGetAsciiTests.cs]: Nit: could be ```Assert.IsType<ArgumentException>(ex);```

**Comment by tarekgh** [comment]
@dotnet-bot test innerloop Debian8.4 Debug
@dotnet-bot test innerloop Debian8.4 Release
@dotnet-bot test innerloop Fedora23 Debug
@dotnet-bot test innerloop Fedora23 Release
@dotnet-bot test innerloop OpenSUSE13.2 Debug
@dotnet-bot test innerloop OpenSUSE13.2 Release
@dotnet-bot test innerloop OpenSUSE42.1 Debug
@dotnet-bot test innerloop OpenSUSE42.1 Release
@dotnet-bot test outerloop Ubuntu16.04 Release
@dotnet-bot test outerloop Ubuntu16.10 Debug

**Comment by tarekgh** [comment]
test Innerloop Debian8.4 Debug Build and Test please

**Comment by Clockwork-Muse** [comment]
Hmmm... should we pull out Windows/OSX to a separate test?  To make it more obvious we expect certain behavior from that platform?

**Comment by tarekgh** [comment]
test Innerloop Debian8.4 Release Build and Test please
test  Innerloop Fedora23 Debug Build and Test please
test  Innerloop Fedora23 Release Build and Test please
test Innerloop OpenSUSE42.1 Release Build and Test please
test  Innerloop Ubuntu14.04 Release Build and Test please

**Comment by tarekgh** [comment]
test Innerloop Debian8.4 Debug Build and Test please

**Comment by tarekgh** [comment]
@Clockwork-Muse 

> Hmmm... should we pull out Windows/OSX to a separate test? To make it more obvious we expect certain behavior from that platform?

The test case already small enough and handling the Windows and OSX case and it is clear too. so I don't think we need to split it (at least for now)

**Pull request merged**